woocommerce/woocommerce-admin
# Update Search component to use SelectControl

Fixes #2834

Replaces the `Autocomplete` component in `Search` with `SelectControl`.

### Screenshots
<img width="551" alt="Screen Shot 2019-10-15 at 7 26 27 PM" src="https://user-images.githubusercontent.com/10561050/66834788-e6eafe00-ef90-11e9-9cc6-984d9acac3b6.png">
<img width="589" alt="Screen Shot 2019-10-15 at 7 23 24 PM" src="https://user-images.githubusercontent.com/10561050/66834790-e6eafe00-ef90-11e9-91c6-50cff9ca75f6.png">

### Detailed test instructions:

1.  Test all instances of the `Search` component.
2.  Make sure options are fetched correctly for each.
3.  Make sure that advanced filters continue to work as expected.
4.  Check that styling of the `Search` component matches existing styling.
